data/quickstarts/deploy-watson-model-quickstart.yaml

```
@@ -0,0 +1,80 @@
+kind: ConsoleQuickStart
+metadata:
+  name: build-deploy-watson-model
+spec:
+  displayName: Deploying a Model with Watson Studio
+  durationMinutes: 15
+  icon: 'images/IBM Logo.png'
```

We have local versions of the icons for now:
```suggestion
  icon: 'images/ibm.svg'
```